### ISSUE TITLE
Sample Configuration not working

### ISSUE THREAD

**Issue by pankajmalkani** [souvikdc9/aadhaarapi.net#13]
Hello,

in Uidai.Aadhaar.Sample, Microsoft.Extensions.Configuration.ConfigurationBuilder().Bind(options) function is not working or found.
Actually Bind(options) is not found in Microsoft.Extensions.Configuration.ConfigurationBuilder class at all,
Please let me know ASAP

**Comment by ghost** [comment]
Yes, it may be possible, if you are targeting newer version of .NET Core. Many NuGet packages and classes changed since early stages of .NET Core. KIndly find the recommended way to bind the configuration file in ASP.NET Core docs.
